modules/samtools/stats/main.nf

```diff
@@ -8,7 +8,7 @@ process SAMTOOLS_STATS {
         'quay.io/biocontainers/samtools:1.15.1--h1170115_0' }"
 
     input:
-    tuple val(meta), path(input), path(input_index)
+    tuple val(meta), path(bam), path(bai)
```

Comment:
Actually as input can be `cram`, we prefer to keep `input` here instead of `bam`

Reply:
I was trying to come up with a bit a standard naming scheme here, since it was a bit alle over the place and the nf-core recommendations are to use `bam`.

I do agree something like `reads` would be better suited, but then that'd need to be changed across all modules. (I like uniformity in my code)

Reply:
I like uniformity too.
I think we more or less have agreed on the notion that reads is good for unmapped reads.
For mapped reads, I'd go for either bam or cram depending on the file type, but when both are possible, we went for a more generic `input`

Reply:
anyhow, we can continue in #1880